program: train_agent.py
method: bayes
metric:
  name: validation/avg_episode_reward
  goal: maximize
project: Model-based_RL_in_Teleoperation
entity: ge62meq-technical-university-of-munich        # W&B username
parameters:
  learning_rate:
    distribution: log_uniform_values
    min: 1e-5  # 0.00001
    max: 1e-3  # 0.001
  gamma:
    distribution: uniform
    min: 0.90
    max: 0.999
  tau:
    distribution: uniform
    min: 0.001
    max: 0.01
  batch_size:
    distribution: categorical
    values: [256, 512]
  action_penalty_weight:
    distribution: log_uniform_values
    min: 1e-3
    max: 1.0
command:
  - ${env}
  - python3
  - ${program}
  - --config=4
  - --seed=50
  - --randomize-trajectory
  # W&B will automatically add --learning_rate=..., --gamma=..., etc.